holoviews/core/layout.py

```diff
@@ -62,8 +62,8 @@ class Empty(Dimensioned, Composable):
 
     group = param.String(default='Empty')
 
-    def __init__(self):
-        super().__init__(None)
+    def __init__(self, **params) -> None:
```

Comment:
We have not started adding type hints to HoloViews. I'm in favor but that should be a project level decision.

```suggestion
    def __init__(self, **params):
```